apache/helpers.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
I suppose helpers.php should be added to the root folder and a line added in update.sh ?

Reply:
Yes of course, I don't understand why I didn't done this yesterday.
Seems logic to add this file in the creation script instead of deal with manually.

Now its done !